cmd/gitops/ui/run/cmd.go

```diff
@@ -72,6 +73,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&options.ProfileCacheLocation, "profile-cache-location", "/tmp/helm-cache", "the location where the cache Profile data lives")
 	cmd.Flags().StringVar(&options.WatcherHealthzBindAddress, "watcher-healthz-bind-address", ":9981", "bind address for the healthz service of the watcher")
 	cmd.Flags().StringVar(&options.WatcherMetricsBindAddress, "watcher-metrics-bind-address", ":9980", "bind address for the metrics service of the watcher")
+	cmd.Flags().StringVar(&options.NotificationControllerAddress, "notification-controller-address", "http://notification-controller", "the address of the notification-controller running in the cluster")
```

Comment:
How about defaulting this to something like...

`http://notification-controller.flux-system.svc.cluster.local./` ?

Reply:
You probably want to read https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ 

That'll help you work out what the right answer is :-)

Reply:
I'm aware of that document, thanks. :) But doesn't flux set up dedicated service accounts for each controller?

Reply:
In any case, I can set that, flux has probably some custom coredns rules to get that to work anyways.

Reply:
`http://notification-controller./`